### PR TITLE
renamed the package, i.e., resolve #67

### LINKED ISSUE #67: Rename allensd.ipfx-> allensdk_ipfx

### PR DESCRIPTION
**Pull request by sgratiy**

### CODE REVIEW

File: ipfx/bin/run_pipeline.py [hunk 1]
```diff
@@ -61,12 +61,12 @@ def run_pipeline(input_nwb_file,
...
-    #                   qc_fig_dir,
-    #                   se_output["sweep_features"],
-    #                   fx_output)
-    # logging.info("Visualized results!")
+    run_visualization(input_nwb_file,
+                      stimulus_ontology_file,
+                      qc_fig_dir,
+                      se_output["sweep_features"],
+                      fx_output)
+    logging.info("Visualized results!")
```
Comment (t-b) [inline ipfx/bin/run_pipeline.py]: @sgratiy If this is an intentional change put it into a separate commit.

File: ipfx/bin/run_pipeline_nwb.py [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
-import allensdk.ipfx.stimulus as stm
-import allensdk.ipfx.qc_protocol as qcp
+import ipfx.stimulus as stm
+import ipfx.qc_protocol as qcp
 import allensdk.core.json_utilities as ju
 import os.path
 from run_pipeline import run_pipeline
 
-stimulus_ontology_file = eds.DEFAULT_STIMULUS_ONTOLOGY_FILE
+stimulus_ontology_file = stm.DEFAULT_STIMULUS_ONTOLOGY_FILE
```
Comment (t-b) [inline ipfx/bin/run_pipeline_nwb.py]: @sgratiy If this is an intentional change put it into a separate commit.

**Comment by t-b** [comment]
- [ ] There is a useless merge commit on top. I tend to get rid of these by rebasing against master. But I don't know what workflow you in that repo here. Avoiding a merge commit also makes the chronological review with `git log --reverse  -p master..` possible.

**Comment by dyf** [comment]
Thanks for the helpful suggestions @t-b.  In the interest of getting this change made so other work can progress, I'm going to recommend @sgratiy go ahead and merge this and we adopt better hygiene moving forward.

**Comment by t-b** [comment]
@dyf Fine!
